examples/3d-tiles/tile-3d-layer.js

```diff
@@ -289,13 +306,43 @@ export default class Tile3DLayer extends CompositeLayer {
     // tileHeader.userData = {gltfObjects};
   }
 
+  _calculatePositionsOffset(tileHeader) {
+    const {center} = tileHeader.boundingVolume;
+    const {distanceScales} = this.context.viewport;
+
+    const {metersPerPixel, pixelsPerDegree} = distanceScales;
+
+    const {pointsCount} = tileHeader.userData;
+    const {positions} = tileHeader.content;
+
+    for (let i = 0; i < pointsCount; i++) {
+      const point = new Vector3(positions[i * 3], positions[i * 3 + 1], positions[i * 3 + 2]);
+
+      Ellipsoid.WGS84.cartesianToCartographic(point, point);
+      const offset = [point[0] - center[0], point[1] - center[1], point[2] - center[2]];
+
+      positions[i * 3] = metersPerPixel[0] * pixelsPerDegree[0] * offset[0];
+      positions[i * 3 + 1] = metersPerPixel[1] * pixelsPerDegree[1] * offset[1];
+      positions[i * 3 + 2] = offset[2];
+    }
+  }
+
+  _isWGS84(tileHeader) {
```

Comment:
Note that this function in current form is not considering tile transforms. So as written, this function will only detect if the coordinates are in WGS84 before transforms are applied.

I.e. there could be a transform that takes local offsets and turns them into big numbers (WGS84).

By applying the tile transform before the check, you should be able to determine accurately if the tile positions are in WGS84 or local coordinates.